helm/opendistro-es/values.yaml

```
@@ -557,3 +557,21 @@ curator:
       loglevel: INFO
       logformat: default
       blacklist: ['elasticsearch', 'urllib3']
+
+configurer:
+  enabled: false
+
```

Considering if we should add an option that enables or disables the inital index creation. Might come in handy in restore situations not to create the inital indexes.
```suggestion

  createIndices: true # or createIndexes, not sure which one to use

```

---

Added :slightly_smiling_face: